### ISSUE TITLE
Text min/max validator - count multi-byte character as one

### ISSUE THREAD

**Issue by serkan-bayram** [pocketbase/pocketbase#3735]
I have this input for my field that has "max length" and it throws an error: 

![image](https://github.com/pocketbase/pocketbase/assets/68502470/04d2334f-68eb-4d1c-ba4a-3586ff91b797)

![image](https://github.com/pocketbase/pocketbase/assets/68502470/d4fd1d50-46a2-4c73-b3b4-cb610292cde0)

```js
{
  "errorMessage": "comment: Must be less than 280 character(s)."
}
``` 

so, is this a known issue or there won't be configurations for these cases?

**Comment by ganigeorgiev** [comment]
Hm, this is side-effect of Go's `len()` applied on non-ascii characters (in Go a single character could be 1-4 bytes).

I'll submit a fix later this weekend to cast the text value to `[]rune` to properly count the multibyte characters.

**Comment by ganigeorgiev** [comment]
I've submitted a small fix in `master`. It will be available with the next minor v0.19.4 release sometime later this weekend.
